learning/templates/index.html

```diff
@@ -16,11 +13,19 @@
 
     {% else %}
 
-        <div id="site-title" class="animate-reveal animate-first">Fiszkly</div>
-        <div id="site-desc" class="animate-reveal animate-second">Let's learn something from your flashcards</div>
-    
-        <div id="homepage"></div>
-        <script src="/static/frontend/home_page.js"></script>
+        <div class="box">
+            <div id="site-title" class="animate-reveal animate-first">Fiszkly</div>
+            <div id="site-desc" class="animate-reveal animate-second">Let's learn something from your flashcards</div>
```

Comment:
Wydaje mi się, że nie to nie powinno być w sekcji learning - albo w base.html żebyśmy handlowali niezalogowanych użytkowników, albo w accounts

Reply:
najlepiej w base.html